### PR TITLE
Release 0.1.6

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,18 @@
 Changes
 =======
 
+0.1.6
+-----
+
+* Add the ``--host`` option to CLI.
+  It corresponds to ``flask run``'s ``--host`` option.
+* When serving (with ``serve`` or ``freeze --serve``), Elsa now listens on
+  Flask's uses default address, ``127.0.0.1``. The server is not visible
```
Comment (hroncok) [inline CHANGELOG.rst:10]:
```suggestion
  Flask's default address, ``127.0.0.1``. The server is not visible
```

or

```suggestion
  Flask's used default address, ``127.0.0.1``. The server is not visible
```

Comment (encukou) [reply CHANGELOG.rst]: Oops! Thanks!

**Review by hroncok — APPROVED** [review]

**Comment by hroncok** [comment]
https://pypi.org/project/elsa/0.1.6/

**Comment by encukou** [comment]
Thank you!

**Pull request merged**